frame/support/src/traits/tokens/currency.rs

```diff
@@ -199,3 +199,90 @@ pub trait Currency<AccountId> {
 		balance: Self::Balance,
 	) -> SignedImbalance<Self::Balance, Self::PositiveImbalance>;
 }
+
+impl<AccountId> Currency<AccountId> for () {
```

Comment:
```suggestion
#[cfg(feature = "std)]
impl<AccountId> Currency<AccountId> for () {
```

to prevent accidental use in production?